src/Adapter/Customer/CommandHandler/BulkDeleteCustomerHandler.php

```diff
@@ -56,5 +58,17 @@ public function handle(BulkDeleteCustomerCommand $command)
             $customer->deleted = true;
             $customer->update();
         }
+
+        PrestaShopLogger::addLog(
+            Context::getContext()->getTranslator()->trans(
+                'Customers bulk deleted',
+                [],
+                'Admin.Advparameters.Notification'
+            ),
+            1,
+            null,
+            'Customer',
+            $customer->id
```

Comment:
The variable `$customer` is not available in this context. Maybe put this call on the loop so we keep track of all the customer's ids that have been deleted?